packages/shorebird_cli/test/src/commands/patch/patch_aar_command_test.dart

```diff
@@ -471,13 +480,16 @@ Did you forget to run "shorebird init"?''',
       const otherRevision = 'other-revision';
       when(() => flutterRevisionProcessResult.stdout).thenReturn(otherRevision);
       final tempDir = setUpTempDir();
+      final flutterDir =
+          runWithOverrides(() => ShorebirdEnvironment.flutterDirectory.path);
```

Comment:
It took me a bit to figure out that this was necessary. It's possible that this is just a bad test though – we probably shouldn't just be duplicating the code we're testing here.